charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.56.0
+version: 3.56.1
```

Comment:
could you bump the minor version instead?

Reply:
Sure 👍 
```diff
/CHANGELOG.md b/charts/datadog/CHANGELOG.md
index 0a0e286..65d14dc 100644
--- a/charts/datadog/CHANGELOG.md
+++ b/charts/datadog/CHANGELOG.md
@@ -1,6 +1,6 @@
 # Datadog changelog

-## 3.56.1
+## 3.57.0

 * Set default `Agent` and `Cluster-Agent` version to `7.51.0`.

diff --git a/charts/datadog/Chart.yaml b/charts/datadog/Chart.yaml
index 9716a07..b44e8fc 100644
--- a/charts/datadog/Chart.yaml
+++ b/charts/datadog/Chart.yaml
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.56.1
+version: 3.57.0
 appVersion: "7"
 description: Datadog Agent
 keywords:
diff --git a/charts/datadog/README.md b/charts/datadog/README.md
index 0a85f11..6a3cd34 100644
--- a/charts/datadog/README.md
+++ b/charts/datadog/README.md
@@ -1,6 +1,6 @@
 # Datadog

-![Version: 3.56.1](https://img.shields.io/badge/Version-3.56.1-informational?style=flat-square) ![AppVersion: 7](https://img.shields.io/badge/AppVersion-7-informational?style=flat-square)
+![Version: 3.57.0](https://img.shields.io/badge/Version-3.57.0-informational?style=flat-square) ![AppVersion: 7](https://img.shields.io/badge/AppVersion-7-informational?style=flat-square)

 [Datadog](https://www.datadoghq.com/) is a hosted infrastructure monitoring platform. This chart adds the Datadog Agent to all nodes in
 your cluster via a DaemonSet. It also optionally depends on the [kube-state-metrics chart](https://github.com/prometheus-community/helm
-charts/tree/main/charts/kube-state-metrics). For more information about monitoring Kubernetes with Datadog, please refer to the [Datado
g documentation website](https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/).

```